osf/models/user.py

```diff
@@ -504,6 +504,12 @@ def csl_name(self, node_id=None):
                 'given': csl_given_name,
             }
 
+    @property
+    def osfstorage_region(self):
+        from addons.osfstorage.models import Region
+        user_settings = self.get_addon('osfstorage')
+        return Region.objects.get(id=user_settings.region_id)
```

Comment:
This breaks because `region_id` isn't an attribute.

```python
AttributeError: 'UserSettings' object has no attribute 'region_id'
```

I think you meant `default_region_id` here. But see my previous comment: this property might not be necessary.

Reply:
Yep, I definitely meant `default_region_id`.